docs/modules/ROOT/pages/static-site-generators.adoc

```diff
@@ -0,0 +1,94 @@
+= Static Site Generators
+
+Asciidoctor fundamentally converts one source document into one output document.
+To convert more than one document, you need some amount of infrastructure.
+Simple situations may find it convenient to use built-in Asciidoctor capabilities or scripts, but are unlikely to be appropriate for any but the smallest sites.
+Static site generators organize the conversion of multiple source documents into a functioning static web site.
+
+== What does a Static Site Generator (ssg) do?
```

Comment:
We should either use "Static Site Generator" everywhere or use "standard casing" here since the term is used in a section title

Reply:
I'm not sure what "standard casing" is.  Except for the doctitle, I've changed to lowercase 'static site generator' everywhere and the only 'SSG' is after the first occurrence.

Reply:
Sorry, the right term seems to be sentence case:

> Sentence case is the conventional way of using capital letters in a sentence or capitalizing only the first word and any proper nouns.

```suggestion
== What does a static site generator do?
```